README.md

```diff
@@ -148,18 +148,17 @@ We provide a pre-built docker container for habitat-api and habitat-sim, refer t
 We highly recommend installing a [miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/distribution/#download-section) environment (note: python>=3.6 is required). Once you have Anaconda installed, here are the instructions.
 
 
-1. Clone this github repository. 
+1. Clone this github repository.
    ```bash
-   git clone https://github.com/facebookresearch/habitat-sim.git
+   # Checkout the latest stable release
+   git clone --branch stable git@github.com:facebookresearch/habitat-sim.git
```

Comment:
Maybe just keep the checkout?  That doesn't feel really any different and then we don't have to maintain that branch :)

Reply:
@erikwijmans that's not a branch, that is a tag:
`git clone --branch <tag_name> <repo_url>`

Reply:
Cool, didn't know git would do that!